api/spec/v1.yaml

```diff
@@ -2303,6 +2303,16 @@ components:
           type: object
           description: The method call body. May be null if the transaction was malformed.
           example: {"address": "t1mAPucIdVnrYBpJOcLV2nZoOFo=", "data": RBo+cAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=", "value": ""}
+        is_likely_native_token_transfer:
```

Comment:
should we add this field to `required`?

Reply:
Maybe. I do have arguments for keeping it optional, but they are relatively weak: 1) We currently just omit it if it's `false`. 2) It's such an unusual field (in that it's a heuristic, rather than just restructured blockchain contents) that I kinda like keeping the door ajar to removing the field later if desired.

I don't need much of a push to promote it to `required` later.